api/src/main/java/bio/terra/tanagra/app/controller/UnderlaysApiController.java

```diff
@@ -60,12 +60,35 @@ public ResponseEntity<ApiListUnderlaysResponse> listUnderlays(
                     .map(UnderlaysApiController::convert)
                     .collect(Collectors.toList()))
             .nextPageToken(page.nextPageToken());
+
     return ResponseEntity.ok(response);
   }
 
   private static ApiUnderlay convert(Underlay underlay) {
     List<String> entityNames =
         underlay.entities().keySet().stream().sorted().collect(Collectors.toList());
-    return new ApiUnderlay().name(underlay.name()).entityNames(entityNames);
+
+    String columns = "[" +
```

Comment:
There's a step missing here and maybe I wasn't clear enough when we discussed this before. This data shouldn't be hardcoded here, or anywhere else, in the code. Instead, it should be in the underlays (e.g. aou_synthetic.yaml) and read along with the rest of the information about the underlay like the entities. That path goes through underlay.proto, UnderlayConversion.java, and Underlay.java.